src/helpers.js

```diff
@@ -17,10 +18,14 @@ export const assign = Object.assign || function (target, source) {
   return target
 }
 
+export function splitEnv(contents: string) {
+  return contents.split(contents.includes('\0') ? '\0' : '\n')
```

Comment:
We are using python splitting when it's available but when it's not we fallback to env, and we still need to split the env output with `\n`